packages/db/src/core/load-file.ts

```diff
@@ -31,7 +28,6 @@ export async function resolveDbConfig({
 	const integrationDbConfigPaths: Array<{ name: string; configEntrypoint: string | URL }> = [];
 	const integrationSeedPaths: Array<string | URL> = [];
 	for (const integration of integrations) {
-		if (!isDbIntegration(integration)) continue;
```

Comment:
It was needed as a type guard to turn `integration` from `AstroIntegration` to `AstroDbIntegration`. That guard is no longer needed since it is a single augmented type now.